src/data/PastEventData.json

```diff
@@ -1,46 +1,67 @@
 {
-  "data": [
+  "EventData": [
```

Comment:
Google JSON Style Guide (recommendations for building JSON APIs at Google), It recommends that: Property names must be camelCased